### ISSUE TITLE
Error: return template withScript in subdir

### ISSUE THREAD

**Issue by SatWiz** [dart-bridge/dart-bridge.github.io#6]
Hi Emil, I am not sure this is a Bridge issue or dart's.

I have defined this router:

```
  routes(Router router) {
    router.get('/test/:id', (shelf.Request request, {String id}) {
      print(id);
      return template('index', withScript: 'test');
    });
  }
```

It is calling this script at /web/test/test.dart:

```
import 'package:bridge/tether_client.dart';
import 'package:app/client.dart';

main() async {
  // Connect the tether
  await globalTether();

  // Register shared data structures
  registerSharedStructures(tether);
}
```

When calling the page as http://localhost:1337/test/1234 I having these errors:

```
GET http://localhost:1337/test/packages/bridge/tether_client.dart ...
An error occurred loading file: package:bridge/tether_client.dart ...
GET http://localhost:1337/test/packages/app/client.dart  ...
An error occurred loading file: package:app/client.dart ...
```

It looks likes the libraries can't be found under /test. But they should not be under /test or should I import it differently under test.dart?

**Comment by emilniklas** [comment]
Ah! This is certainly a bug.

The script tag generated should be `<script src='/test.dart'>`, but right now it's `<script src='test.dart'>`.

I will fix this, but a temporary fix is for you to say `router.get('test', () => template('test', withScript: '../test'))`

The the url for your src import will be `/test/../test.dart` which evaluates to just `/test.dart`

**Comment by emilniklas** [comment]
Fixed as of `1.0.0-alpha5`
